docs/USAGE-GUIDE.md

```diff
@@ -325,12 +325,14 @@ Call `s2n_print_stacktrace()` to print your stacktrace.
 **Note:** Enabling stacktraces can significantly slow down unit tests, causing failures on tests (such as `s2n_cbc_verify`) that measure the timing of events.
 
 
-## Initialization and teardown
+## Initialization and Teardown
 
 The s2n-tls library must be initialized with `s2n_init()` before calling most library functions. `s2n_init()` MUST NOT be called more than once, even when an application uses multiple threads or processes. To clean up, `s2n_cleanup()` must be called from every thread or process created after `s2n_init()` was called.
 
 Initialization can be modified by calling `s2n_crypto_disable_init()` or `s2n_disable_atexit()` before `s2n_init()`.
 
+An application can override s2n-tls’s internal memory handling functions by calling `s2n_mem_set_callbacks` before calling s2n_init.
```

Comment:
Would "internal memory allocation" or "internal memory management" be more descriptive?